blog/2025-02-11-announcing-buildernet-v1.2.md

```diff
@@ -0,0 +1,160 @@
+---
+title: Announcing BuilderNet v1.2
+description: We are excited to announce the release of BuilderNet v1.2! This release introduces fully automatic node bootstrapping, improved system management capabilities and authentication, and lays the groundwork for a more flexible, container-based architecture.
```

Comment:
"We are excited to release BuilderNet v1.2, a significant upgrade that streamlines operator onboarding and enhances the security and performance of the BuilderNet infrastructure."